src/app/public/modules/select-field/select-field-picker.component.html

```diff
@@ -9,7 +9,10 @@
           [data]="data"
           [selectedIds]="selectedIds"
           (selectedIdsChange)="onSelectedIdsChange($event)">
-          <sky-list-toolbar>
+          <sky-list-toolbar
+            [inMemorySearchEnabled]="context?.inMemorySearchEnabled"
+            (searchApplied)="onSearchApplied($event)"
```

Comment:
👆 Neither of these properties exist in `list-builder` today. We'll have to add them.